integration-tests/taquito-conversion-utils.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { LocalForger } from "@taquito/local-forging";
+import { toForge } from "@taquito/taquito";
+import { CONFIGS } from "./config";
+
+CONFIGS().forEach(({ lib, setup, createAddress }) => {
+  const Tezos = lib;
+  const forger = new LocalForger();
+
+  describe(`Test taquito common util functions in context`, () => {
+    beforeEach(async (done) => {
+      await setup();
+      done();
+    })
+
+    it('Verify toPreaplyParams returns executable params for preapplyOperations', async (done) => {
```

Comment:
I think these tests should also assert the values being output, and we should also see if they're able to be passed to its respective recipient functions

Reply:
this one was addressed. they are before the if block but dont show on this